controllers/schools.js

```diff
@@ -189,6 +196,25 @@ const getHandler = async (req, res) => {
       ];
     });
 
+    if(USER_MIGRATION_ENABLED){
+      const migrationHead = ['Migration gestartet', 'Migration verpflichtend', 'Migration abgeschlossen', 'Migration final beendet', 'Login-System'];
+      head.splice(5, 0, ...migrationHead);
+
+      const migrationBody = schools.data.map((item) => {
+        return [
+          item.oauthMigrationStart ? getDateFormat(item.oauthMigrationStart) : '',
+          item.oauthMigrationMandatory ? getDateFormat(item.oauthMigrationMandatory) : '',
+          item.oauthMigrationFinished ? getDateFormat(item.oauthMigrationFinished) : '',
+          Date.now() >= new Date(item.oauthMigrationFinalFinish).getTime() ? getDateFormat(item.oauthMigrationFinalFinish) : '',
+          item.systems.map(system => system.alias).join(',') || '',
+        ];
+      });
+
+      body.forEach((item, i) => {
+        item.splice(5, 0, ...migrationBody[i]);
+      });
+    }
+
```

Comment:
Oh kids, why should it be so complicated. Please move your logic to function and call it in line 196

Reply:
Fixed